formats/polus-tabular-to-feather-plugin/src/main.py

```diff
@@ -0,0 +1,201 @@
+logger.setLevel(POLUS_LOG)
+
+#Set number of processors for scalability
+NUM_CPUS = max(1, cpu_count() // 2)
+
+def csv_to_df(file):
+    """Convert csv into datafram or hdf5 file. Copied partially from Nicholas-Schaub polus-csv-to-feather-plugin
+            
+            Args:
+                file (str): Path to input file.
+                filePattern (str): extension of file to convert.
+                
+            Returns:
+                Vaex dataframe
+                
+    """
+    logger.info('CSV CONVERSION: Copy csv file into outDir for processing...')
+    file_name = Path(file).stem
+    output = file_name + ".csv"
+    outputfile = os.path.join(outDir, output)
+    shutil.copyfile(file, outputfile)
+    
+    logger.info('CSV CONVERSION: Checking size of csv file...')
+    # Open csv file and count rows in file
+    with open(outputfile,'r', encoding='utf-8') as fr:
+        ncols = len(fr.readline().split(','))
+        
+    chunk_size = max([2**24 // ncols, 1])
+    logger.info('CSV CONVERSION: # of columns are: ' + str(ncols))
+        
+    # Convert large csv files to hdf5 if more than 1,000,000 rows
+    logger.info('CSV CONVERSION: converting file into hdf5 format')
+    df = vaex.from_csv(outputfile,convert=True,chunk_size=chunk_size)
+    return df
+
+def binary_to_df(file,filePattern):
+    """Convert any binary formats into vaex dataframe (.arrow, .parquet, .hdf5, or .fits).
+            
+            Args:
+                file (str): Path to input file.
+                filePattern (str): extension of file to convert.
+                
+            Returns:
+                Vaex dataframe.
+                
+    """
+    binary_patterns = [".fits", ".arrow", ".parquet", ".hdf5"]
+
+    logger.info('BINARY FILE: Scanning directory for binary file pattern... ')
+    if filePattern in binary_patterns:
+        #convert hdf5 to vaex df
+        df = vaex.open(file)
+        return df
+    else:
+        raise FileNotFoundError('No supported binary file extensions were found in the directory. Please check file directory again.' )
+    
+def fcs_to_feather(file,outDir):
+    """Convert fcs file to csv. Copied from polus-fcs-to-csv-converter plugin.
+            
+        Args:
+            file (str): Path to the directory containing the fcs file.
+            outDir (str): Path to save the output csv file.
+                
+        Returns:
+            Converted csv file.
+                
+    """
+    file_name = Path(file).stem
+    feather_filename = file_name + ".feather"
+    logger.info('FCS CONVERSION : Begin parsing data out of .fcs file' + file_name)
+    
+    #Use fcsparser to parse data into python dataframe
+    meta, data = fcsparser.parse(file, meta_data_only=False, reformat_meta=True)
+    
+    #Export the fcs data to vaex df
+    logger.info('FCS CONVERSION: converting data to vaex dataframe...')
+    df = vaex.from_pandas(data)
+    logger.info('writing file...')
```

Comment:
Both here and two lines down, you should use the same logging pattern as what you did above. Ideally, this would look like:

```python
logger.info('fcs_to_feather: Writing feather file...')
```